api/logs/s3/s3_test.go

```diff
@@ -0,0 +1,28 @@
+package s3
+
+import (
+	"net/url"
+	"os"
+	"testing"
+
+	logTesting "github.com/fnproject/fn/api/logs/testing"
+)
+
+func TestS3(t *testing.T) {
+	minio := os.Getenv("MINIO_URL")
+	if minio == "" {
+		t.Skip()
```

Comment:
worth adding a reason for skipping tests

Reply:
looks pretty obvious to me, this is set up in the test script